contrib/centos/aziot-identity-service.spec

```diff
@@ -174,7 +174,7 @@ fi
 %attr(400, aziottpm, aziottpm) %{_sysconfdir}/aziot/tpmd/config.toml.default
 %attr(700, aziottpm, aziottpm) %dir %{_sysconfdir}/aziot/tpmd/config.d
 
-%attr(400, root, root) %{_sysconfdir}/aziot/config.toml.template
+%attr(700, root, root) %{_sysconfdir}/aziot/config.toml.template
```

Comment:
Wait, shouldn't this be 600?

Reply:
Ah yes, I fixed it in iotedge but forgot to fix it here. Fixed now.